src/pages/home/report/ReportActionCompose/ComposerWithSuggestions/ComposerWithSuggestions.js

```diff
@@ -220,16 +222,42 @@ function ComposerWithSuggestions({
                     debouncedUpdateFrequentlyUsedEmojis();
                 }
             }
-            const newCommentConverted = convertToLTRForComposer(newComment);
+
+            let newCommentConverted = convertToLTRForComposer(newComment);
+
+            const moveCursorToEndOfLine = (commentLength) => {
+                setSelection({
+                    start: commentLength + 1,
+                    end: commentLength + 1,
+                });
+            };
+
+            if (firstChange && commentRef.current === '' && !['@'].includes(newComment)) {
+                if (commentRef.current !== newComment) {
+                    setValue(convertToLTR(newComment));
+                    moveCursorToEndOfLine(newComment.length);
+                }
+                setFirstChange(false);
+            } else if (commentRef.current !== newComment) {
+                newCommentConverted = newComment.length <= 1 ? newCommentConverted.replace(/\u2066/g, '') : newCommentConverted;
+                setValue(newCommentConverted);
+                moveCursorToEndOfLine(newComment.length);
+                setFirstChange(false);
+            }
+
             const isNewCommentEmpty = !!newCommentConverted.match(/^(\s)*$/);
             const isPrevCommentEmpty = !!commentRef.current.match(/^(\s)*$/);
 
             /** Only update isCommentEmpty state if it's different from previous one */
             if (isNewCommentEmpty !== isPrevCommentEmpty) {
                 setIsCommentEmpty(isNewCommentEmpty);
+                if (isNewCommentEmpty) {
+                    setFirstChange(true);
```

Comment:
It's strange to see this being set here, but maybe it's only confusing because of the variable name.

Reply:
My bad, we need to specify when the composer is empty because when unicode is present It is not really empty but it detects like it does.